subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/EnableFileSystemWatchingIntegrationTest.groovy

```diff
@@ -78,11 +86,53 @@ class EnableFileSystemWatchingIntegrationTest extends AbstractFileSystemWatching
         when:
         run("assemble", commandLineOption, "--info")
         then:
-        outputContains(expectedMessage)
+        outputContains(expectedEnabledMessage)
+        outputContains(expectedActiveMessage)
+
+        where:
+        commandLineOption | expectedEnabledMessage | expectedActiveMessage
+        "--watch-fs"      | ENABLED_MESSAGE        | ACTIVE_MESSAGE
+        "--no-watch-fs"   | DISABLED_MESSAGE       | INACTIVE_MESSAGE
+    }
+
+    @Unroll
```

Comment:
You don't need `@Unroll` any more since we are using Spock 3 now.

Reply:
Let's keep the file consistent for now, and remove `@Unroll` everywhere separately.